packages/components/forms/src/FormLabel/FormLabel.tsx

```diff
@@ -0,0 +1,68 @@
+import { cx } from 'emotion';
+import React, { forwardRef } from 'react';
+import type { ReactNode } from 'react';
+import type { PolymorphicComponentProps } from '@contentful/f36-core';
+import { getFormLabelStyles } from './FormLabel.styles';
+import { useFormControl } from '../FormControl/FormControlContext';
+
+export type FormLabelInternalProps = {
+  /**
+   * Label value to show
+   */
+  children: ReactNode;
+  /**
+   * Whether or not the associated input element is required
+   *
+   * @default false
+   */
+  isRequired?: boolean;
+  /**
+   * Custom text to show in parentheses that gets rendered if the associated
+   * input is required
+   *
+   * @default "required"
+   */
+  requiredText?: string;
+};
+
+export type FormLabelProps = PolymorphicComponentProps<
+  'label',
+  FormLabelInternalProps,
+  'as' | 'ref'
+>;
+
+export const FormLabel = forwardRef<HTMLLabelElement, FormLabelProps>(function (
```

Comment:
It's actually called FormLabel in v3: https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/FormLabel

Reply:
So, basically, I'm just reverting the change that was made in v4

Reply:
But we decided to drop the "Form" prefix in v4

I didn't code that component, I just have a vague memory of this being talked about
and the component was already in v4 named `Label`

Reply:
There are other components being renamed in v4 btw

I know I renamed RelativeDate for example

Reply:
And I just finished reviewing a PR that renamed `RadioButton` to `Radio` and you approved already 👀 

please don't kill me 🙏  🤣

Reply:
I'd only rename components that make sense to rename. `RadioButton` => `Radio` makes sense. 

`FormLabel` to `Label` - I personally doubt.

Reply:
Could you please then talk to the person who renamed the component in the first place before undoing their change?

Or maybe consult the chapter to see what other people think?

Reply:
I'm re-reviewing this PR and I stand by my opinion that this renaming should not be done or we will rename this component every week

and I think the folder should be lowercased like the other folders

Reply:
@gui-santos as agreed with @bgutsol and @denkristoffer, we keep `FormLabel` as we had in v3 for backward compatibility reasons, but we use `FormControl.Label` in all our examples and documentation, because this component should always be used inside `FormControl` section